azure-pipelines.yml

```diff
@@ -0,0 +1,109 @@
+trigger:
+  - master
+
+pr:
+  - master
+
+variables:
+  pytest_extra_flags: ''
+
+jobs:
+
+- job: Linux
+  strategy:
+    matrix:
+      py35-min:
+        conda_env: py35-min
+      py36:
+        conda_env: py36
+      py37:
+        conda_env: py37
+      py36-dask-dev:
+        conda_env: py36-dask-dev
+      py36-zarr-dev:
+        conda_env: py36-zarr-dev
+      # TODO: add numpy-dev to the test-matrix
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - template: ci/azure/unit-tests.yml
+
+- job: MacOSX
+  strategy:
+    matrix:
+      py36:
+        conda_env: py36
+  pool:
+    vmImage: 'macOS-10.13'
+  steps:
+  - template: ci/azure/unit-tests.yml
+
+- job: Windows
+  strategy:
+    matrix:
+      py36:
+        conda_env: py36-windows
+      py37:
+        conda_env: py37-windows
+  pool:
+    vmImage: 'vs2017-win2016'
+  steps:
+  - template: ci/azure/unit-tests.yml
+
+- job: CodeChecks
+  variables:
+    conda_env: py37
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - template: ci/azure/install.yml
+  - bash: |
+      source activate test_env
+      flake8
+    displayName: flake8 lint checks
+  - bash: |
+      source activate test_env
+      mypy .
+    displayName: mypy type checks
```

Comment:
GitHub incorrectly interprets this as part of the `bash` string; confirmed that other parsers correctly treat this as a new item